src/utils/api/lsk/transactions.js

```diff
@@ -74,21 +72,40 @@ export const getSingleTransaction = ({
     }).catch(reject);
 });
 
-/**
- * Calculates the min. transaction fee needed for a transaction
- *
- * @param {object} transaction transaction object
- * @param {number} type transaction type
- * @returns {number} min transaction fee in Beddows
- */
-export const calculateMinTxFee = (
-  transaction, type,
-) => {
-  const fees = findTransactionSizeInBytes({
-    transaction, type,
-  }) * minFeePerByte + transactionTypes.getNameFee(type);
-
-  return fees;
+const txTypeClassMap = {
+  transfer: Lisk.transactions.TransferTransaction,
+  registerDelegate: Lisk.transactions.DelegateTransaction,
+  vote: Lisk.transactions.VoteTransaction,
+};
+
+
+// eslint-disable-next-line max-statements
+export const createTransactionInstance = (rawTx, type) => {
+  const DUMMY_FEE = '18446744073709551615';
+  const DUMMY_SIGNATURE = '204514eb1152355799ece36d17037e5feb4871472c60763bdafe67eb6a38bec632a8e2e62f84a32cf764342a4708a65fbad194e37feec03940f0ff84d3df2a05';
```

Comment:
Are these dummy values planned to be removed? and when?

Reply:
These are necessary to calculate the transaction size. I don't know when we will/can remove them right now.

Reply:
@isalga Their names are somewhat confusing. Those are values of 8 and 64bytes. We need them to. calculate the minimum fee to use as a base, even before the user enters any value in Send form fields.